Dockerfile

```diff
@@ -1,6 +1,6 @@
 FROM python:3.10-alpine as base
 FROM base as builder
-RUN apk add build-base
+RUN apk add build-base libmagic
```

Comment:
The libmagic is not part of the base image, you are adding it to the builder image  so it will not be available in the final one. Add it after line 9